packages/mermaid/src/docs/intro/getting-started.md

```diff
@@ -34,79 +69,137 @@ graph TD
+```
 
-### Saving a Diagram:
+<br />
 
-You may choose any of the methods below, to save it
+#### • Saving a diagram
 
-**We recommend that you save your diagram code on top of any method you choose, in order to make edits and modifications further down the line.**
+There are multiple ways of saving your diagram from the `Actions` section:
+
+- export PNG
+- export SVG
+- export as Markdown
 
 ![Flowchart](./img/Live-Editor-Choices.png)
 
-### Editing your diagrams
+<br />
+
+#### • Editing your diagrams
+
+To edit your diagram, you can copy paste existing Mermaid diagram code into the `Code` section of the `Live Editor`.
 
-Editing is as easy as pasting your **Diagram code**, into the `code` section of the `Live Editor`.
+Or:
 
-### Loading from Gists
+- create a new diagram from scratch
+- use a Sample Diagram from the `Sample Diagrams` section
 
-The Gist you create should have a code.mmd file and optionally a config.json. [Example](https://gist.github.com/sidharthv96/6268a23e673a533dcb198f241fd7012a)
+<br />
 
-To load a gist into the Editor, you can use https://mermaid.live/edit?gist=https://gist.github.com/sidharthv96/6268a23e673a533dcb198f241fd7012a
+#### • Loading from Gists
+
+The Gist you create should have a `code.mmd` file and optionally a `config.json`, similar to this [example](https://gist.github.com/sidharthv96/6268a23e673a533dcb198f241fd7012a).
+
+```note
+To learn about Gists, visit the GitHub documentation page on [Creating gists](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists).
+```
 
-and to View, https://mermaid.live/view?gist=https://gist.github.com/sidharthv96/6268a23e673a533dcb198f241fd7012a
+Once you have created a Gist, copy paste the Gist URL into the respective field in the `Actions` section and click on the `Load Gist` button.
 
-## 2. Using Mermaid Plugins:
+Here is an example of a Gist being loaded into the Editor:
 
-You can generate mermaid diagrams from within popular applications using plug-ins. It can be done in the same way, you would use the Live Editor. Here's a list of [Mermaid Plugins](../ecosystem/integrations-community.md).
+<https://mermaid.live/edit?gist=https://gist.github.com/sidharthv96/6268a23e673a533dcb198f241fd7012a>
 
-**This is covered in greater detail in the [Usage section](../config/usage.md)**
+And, here is the diagram view from the above example:
 
-## 3. Calling the JavaScript API
+<https://mermaid.live/view?gist=https://gist.github.com/sidharthv96/6268a23e673a533dcb198f241fd7012a>
 
-This method can be used with any common web server like Apache, IIS, nginx, node express.
+## 3. Using Mermaid Plugins
 
-You will also need a text editing tool like Notepad++ to generate a .html file. It is then deployed by a web browser (such as Firefox, Chrome, Safari, but not Internet Explorer).
+### Mermaid Plugins
+
+You can generate Mermaid diagrams from within popular applications using plug-ins.
+
```

Comment:
Should we mention the most common use-case of mermaid here, which is probably GitHub's/GitLab's integration?

`````suggestion

For many apps that support markdown (e.g. [GitHub][] and [GitLab][]), you can add mermaid diagrams to markdown by making a `mermaid` code block, like so:

````markdown
The following code-block will be rendered as a mermaid diagram:

```mermaid
flowchart LR
  Hello --> World
```
````

[GitHub]: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams
[GitLab]: https://docs.gitlab.com/ee/user/markdown.html#mermaid

`````

Reply:
💯 we should add these.